src/screens/Authentication/Login.tsx

```diff
@@ -83,22 +151,46 @@ const Login: React.FC = ({}) => {
           <View style={style1.mb20}>
             <Text style={style.textDescription}>Email / Số điện thoại</Text>
             <TextInput
-              style={style.input}
+              style={{
+                flex: 1,
+                minHeight: 50,
+                borderColor: securityUsername === false ? red : grey,
```

Comment:
những biến kiểu boolean thì không check  === true hoặc === false. Tập cách viết ngắn dần cho quen 
```suggestion
                borderColor: !securityUsername ? red : grey,
```